kate/kate_array_ex_2.rb

```diff
@@ -0,0 +1,5 @@
+
+arr = *[24, 78,  39, 5, 92, 18, 44, 13, 54, 7]
+#arr = *(1..10)
+
+result = puts arr.select(&:even?).join(", ")
```

Comment:
rewrite the line 5 of your code to the follows:
result = kate_array_ex_1.rb
puts "#{result}"

By rewriting your code, it makes you code easy to understand.